packages/dds/matrix/src/matrix.ts

```diff
@@ -507,7 +507,7 @@ export class SharedMatrix<T = any>
             default: {
                 assert(content.type === MatrixOp.set, 0x020 /* "Unknown SharedMatrix 'op' type." */);
 
-                const setOp = content as ISetOp<Serializable<T>>;
+                const setOp = content as ISetOp<T>;
```

Comment:
Why?

Reply:
ISetOp<T> uses MatrixItem<T> for its value, and `MatrixItem<T> = Serializable<Exclude<T, null>> | undefined;`

Thus ISetOp should take T not Serializable<T>, since we have no need to apply Serializable twice to the value.

This used to happen to build (due to the compiler incorrectly thinking all Serializable<T> extend T I think), but now it does not due to more restricted type checking with Serializable.

I'm pretty sure the old node was a mistake (needless extra Serializable), but not technically wrong (applying Serializable is idempotent). It just happens to confuse the compiler now, so I made the code what I think was intended.